### ISSUE TITLE
Feature: Add colors to test runner output

### ISSUE THREAD

**Issue by BennieCopeland** [nspec/DotNetTestNSpec#5]
There is no colorization of the contexts and tests like previous test runners.

**Comment by BrainCrumbz** [comment]
Yes, definitely. At the moment runner has not yet implemented colored output, also because there are currently issues with dotnet itself, that prevent it from showing colored text. Cfr. https://github.com/dotnet/cli/issues/1977#issuecomment-248635335

**Comment by amirrajan** [comment]
Aside/long term: If there was a json output (or something equally easy to parse) of 

```csharp
var results = testRunner.Run()
File.Write('output.json', results.ToJsonString())
```

Then that file can be piped to anything to reformat.

There is a [`ConsoleFormatter`](https://github.com/nspec/NSpec/blob/master/sln/src/NSpec/Domain/Formatters/ConsoleFormatter.cs) that has color stuff.

**Comment by BrainCrumbz** [comment]
As an alternative, until state of things changes, we could revert our decision taken on NSpecRunner *not* being shipped for .NET Core targets, and deploy that again.

The only differences would be that executable is not located in `solutionDir\packages\NSpecVersionedDir\tools` . Instead it is located at `userHomeDir\.nuget\packages\NSpecVersionedDir\tools`, and that you run it with something like:

```dos
> dotnet ^
  userHomeDir\.nuget\packages\NSpecVersionedDir\tools\NSpecRunner.dll ^
  yourProject\bin\Debug\yourProject.dll
```

Can this be useful?
